File: TuPAC/HTML_UL/Python/001.py
from lib001 import *

list = HTMLUl()
file = open("index.html", "w")
content = list.convert(
    [
        "Hola Mundo",
        list.convert(
            [
                1,
                [1,2,3]
            ]
        )
    ]
)

file.write(content)
file.close

File: TuPAC/HTML_UL/JS/001.js
function HTMLUl(){
    this.convert = HTMLUlConvert;
}

function HTMLUlConvert(array){
    let html = [];

    for(i in array){
        let element = array[i];

        if(typeof(element) != "object"){
            html.push(`<li>${element}</li>`);
        }
        else{
            html.push(`<li>${this.convert(element)}</li>`);
        }
    }

    return `<ul>${html.join("")}</ul>`;
}